code/gerasimov_dmitry_lab1/solving_quadratic_equation.py

```diff
@@ -0,0 +1,25 @@
+import math
+
+NO_SOLUTION = "No solution"
+
+
+def solving_quadratic_equation(a, b, c):
```

Comment:
That's really inconsistent. Why does this function return a single number in one case, tuple in another case and a string in the third case?